src/collections/_documentation/platforms/php/index.md

```diff
@@ -91,14 +108,29 @@ Sentry\configureScope(function (Sentry\State\Scope $scope): void {
 Tags are key/value pairs assigned to events that can be used for breaking down
 issues or quick access to finding related events.
 
-Most SDKs generally support configuring tags by configuring the scope:
+You can set tags to be added to all events:
 
 ```php
 Sentry\configureScope(function (Sentry\State\Scope $scope): void {
   $scope->setTag('page_locale', 'de-at');
+  
+  // to set multiple key-value pairs at once:
+  $scope->setTags([
+    'page_locale' => 'de-at',
+    'experiment_group' => 'control'
+  ]);
 });
 ```
 
+or per event:
+
+```php
+Sentry\withScope(function (Sentry\State\Scope $scope): void {
+  $scope->setTag('my-tag', 'my value');
+  // or $scope->setTags([ ... ])
+  Sentry\captureException(new \Throwable('my error'))
```

Comment:
`\Throwable` in PHP is an interface, so it cannot be thrown. Please change this code to `new \Exception('...')`

Reply:
✔️